### ISSUE TITLE
Show a helpful error when encountering an access policy violation

### ISSUE THREAD

**Issue by scotttrinh** [edgedb/edgedb-ui#349]
The UI has the ability to run while bypassing access policies, and in fact, most editing that's done directly in the data explorer or query editor are likely admin-level edits that will need to bypass access policies.

In addition to reflecting back the database error, it would be helpful if we would link users directly to the Client Settings and direct them to switch off `enable_access_policies`.

**Comment by gnomical** [comment]
I agree, it took me a bit to realize that the UI was enforcing access policies.
